invenio_rdm_records/records/mappings/v6/rdmrecords/drafts/draft-v4.0.0.json

```diff
@@ -239,10 +239,32 @@
                   "type" : "text"
                 },
                 "lang" : {
-                  "type" : "keyword"
+                  "type": "object",
+                  "properties" : {
+                    "@v" : {
+                      "type" : "keyword"
+                    },
+                    "id" : {
+                      "type" : "keyword"
+                    },
+                    "title" : {
+                      "type" : "object"
```

Comment:
I'm guessing the `dyname: true` is not needed, right? I commented it in the other PR... just checking if applicable or not 👼

Reply:
Indeed, I think we should add the dinamic to add fields dinamically :+1:  Thanks!